File: lab2/zadanie1/zadanie1.c
#include <stdio.h>
int gcd(int a, int b){
	int c;
	while (b > 0){
		c = a%b;
		a = b;
		b = c;
	}
	return a;
}
int main(){
  int a,b,c;
  printf("Podaj pierwszą liczbę: ");
  scanf("%d", &a);
  printf("Podaj drugą liczbę: ");
  scanf("%d", &b);

	a = gcd(a, b);

  printf("%d", a);

}

File: lab2/zadanie1/zadanie1.py
def gcd(a, b):
	while b > 0:
		a, b = b, a%b
	return a

def main():
	a = int(input("Podaj pierwszą liczbę: "))
	b = int(input("Podaj drugą liczbę: "))
	return gcd(a,b )

if __name__ == "__main__":
	print(main())
